code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -478,7 +478,7 @@
 	icon_state = "tile-bluespace"
 	w_class = 3
 	force = 6
-	materials = list(MAT_METAL=937.5)
+	materials = list(MAT_METAL=500)
```

Comment:
I do wonder if this one (and the sepia) should contain something like uranium, plasma, or silver (for the sepia one anyway) as these are kinda special.